src/gui/speedlimitdialog.cpp

```diff
@@ -57,9 +57,9 @@ SpeedLimitDialog::SpeedLimitDialog(QWidget *parent)
     m_ui->setupUi(this);
 
     m_ui->labelGlobalSpeedIcon->setPixmap(Utils::Gui::scaledPixmap(UIThemeManager::instance()->getIcon(QLatin1String("slow_off"))
-                                                , this, style()->pixelMetric(QStyle::PM_SmallIconSize, nullptr, this)));
+                                                , this, style()->pixelMetric(QStyle::PM_LargeIconSize, nullptr, this)));
```

Comment:
Also this should be using `scaledPixmapSvg()`. 
The same to the counterpart icon and also in optionsdialog.cpp.

Reply:
changed to `scaledPixmapSvg`